### ISSUE TITLE
If application message has error, I cannot requestApplication any more

### ISSUE THREAD

**Issue by kikakkz** [linera-io/linera-protocol#1071]
Description:
If the history message of an application has some error, it will report
```
{
  "data": null,
  "errors": [
    {
      "message": "Chain error: Execution error: User application reported an error: Content already exists",
      "locations": [
        {
          "line": 2,
          "column": 3
        }
      ],
      "path": [
        "requestApplication"
      ]
    }
  ]
}
```
when call requestApplication, then cannot requestApplication any more.

Steps:
1  Deploy application
2  Send message which will cause error in application (e.g. in ResPeer, if content is conflict, we will report error 'Content already exists')
3  Call requestApplication again

Expect Result:
requestApplication return successfully.

Actual Result:
requestApplication return the execution error.

**Comment by ma2bd** [comment]
Thanks @kikakkz . Addressing this issue in the correct way (#989) is taking a bit of time (because all the other stuff going on) but I just put out a quick fix #1072 to unblock you.

**Comment by kikakkz** [comment]
> Thanks @kikakkz . Addressing this issue in the correct way (#989) is taking a bit of time (because all the other stuff going on) but I just put out a quick fix #1072 to unblock you.

thanks @ma2bd
